strategies/categorized/src/main/java/org/arquillian/smart/testing/strategies/categorized/TagsParser.java

```diff
@@ -107,5 +106,11 @@ private Annotation findJUnit5TagAnnotation(Annotation annotation) {
         return annotation;
 
     }
+
+    @Override
+    protected boolean isTestMethod(Method method) {
+        return Arrays.stream(method.getAnnotations())
+            .anyMatch(annotation -> annotation.annotationType().getName().equals("org.junit.jupiter.api.Test"));
```

Comment:
I usually refer to do `"org.junit.....".equals(instance)` to avoid NPE

Reply:
Yeah, in the cases when the NPE could occur, I use the same format. In this case, I don't expect that the name of any annotation type could be null :-)
But I can change it if you want...

Reply:
No worries, in fact now that you mention I think that in the annotation is impossible to set a null.